scripts/profiler/merge_data.py

```diff
@@ -3,7 +3,8 @@
 #
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 
-from events import EventsData, Event, EventType
+from processed_events import ProcessedEvents
```

Comment:
Did you test `merge_data` after the changes are introduced?

Reply:
It didn't work for a while after one force push but it has been fixed on Friday and as far as I know it works fine now. Maybe you have worked on old version from Friday morning or earlier?